examples/machine-learning/a4-highgpu-8g/a4high-slurm-blueprint.yaml

```diff
@@ -198,6 +199,11 @@ deployment_groups:
         - protocol: udp
         - protocol: icmp
 
+  # If you are using the Managed Lustre instance below, uncomment this block
```

Comment:
Please move this to be directly next to the Lustre module within the blueprint. The listed order of modules is irrelevant to Terraform but it will be less pront to user-failure if they are next to one another.